### DISCUSSION TITLE
Variables color covered by theme default

### DISCUSSION THREAD

**Discussion by adongden** [izhangzhihao/intellij-rainbow-brackets#2596] [Q&A] [answered]
I changed this color generate to purple, when first open project , it turns to purple, but after file reopen , colors covered by theme default:
## Env
IDEA : 2022.2
rainbow-brackets: 2023.1.3-ij
OS:macOS Monterey 12.5 Apple M2
## First Open
<img width="657" alt="image" src="https://user-images.githubusercontent.com/122472234/217989980-40cbbfb6-081c-4252-9e1c-529dfb4bc21e.png">
## reopen 
<img width="580" alt="image" src="https://user-images.githubusercontent.com/122472234/217990085-75c6d727-88ff-4826-b0ce-58ccd387727b.png">
## brackets config
<img width="705" alt="image" src="https://user-images.githubusercontent.com/122472234/217990210-03d74042-8f1c-438a-9f49-bf58d256fde8.png">
## the theme default
<img width="704" alt="image" src="https://user-images.githubusercontent.com/122472234/217990321-ee076f4a-efd9-4e9f-85e7-346ac027873f.png">

[a comment by an automated account is not shown]

**Comment by izhangzhihao** [comment]
Hi, I changed my config the same as you, but after some restarts of my IDE, I can still see that my brackets and variables are still purple. 

Did your issues resolved or are you still have this issue?


My env:

```
IntelliJ IDEA 2022.2.4 (Ultimate Edition)
Build #IU-222.4459.24, built on November 22, 2022

Runtime version: 17.0.5+7-b469.71 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 12.6.2
GC: G1 Young Generation, G1 Old Generation
Memory: 4096M
Cores: 16
Metal Rendering is ON
Registry:
    ide.intellij.laf.enable.animation=true
    ide.plugins.snapshot.on.unload.fail=true
    ide.tooltip.initialDelay=900
    ide.balloon.shadow.size=0
    scala.erase.compiler.process.jdk.once=false

Non-Bundled Plugins:
    org.nik.presentation-assistant (1.0.10)
    net.sf.opk.avro-schema-support (213.2.1)
    net.seesharpsoft.intellij.plugins.csv (2.19.0)
    com.mallowigi.idea (18.1.0)
    org.jetbrains.kotlin (222-1.7.20-release-201-IJ4167.29)
    com.github.copilot (1.1.33.1980)
    atom-one (1.5.1)
    String Manipulation (9.5.1)
    org.sonarlint.idea (7.1.1.54565)
    com.mallowigi (73.0.0)
    Pythonid (222.4459.24)
    org.intellij.scala (2022.2.13)
    com.intellij.bigdatatools (222.3345.33)
    izhangzhihao.rainbow.brackets.lite (1.0.0)
    izhangzhihao.rainbow.brackets (2023.1.3-ij)

Kotlin: 222-1.7.20-release-201-IJ4167.29
```

**Comment by adongden** [comment]
> Hi, I changed my config the same as you, but after some restarts of my IDE, I can still see that my brackets and variables are still purple.
> 
> Did your issues resolved or are you still have this issue?
> 
> My env:
> 
> ```
> IntelliJ IDEA 2022.2.4 (Ultimate Edition)
> Build #IU-222.4459.24, built on November 22, 2022
> 
> Runtime version: 17.0.5+7-b469.71 x86_64
> VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
> macOS 12.6.2
> GC: G1 Young Generation, G1 Old Generation
> Memory: 4096M
> Cores: 16
> Metal Rendering is ON
> Registry:
>     ide.intellij.laf.enable.animation=true
>     ide.plugins.snapshot.on.unload.fail=true
>     ide.tooltip.initialDelay=900
>     ide.balloon.shadow.size=0
>     scala.erase.compiler.process.jdk.once=false
> 
> Non-Bundled Plugins:
>     org.nik.presentation-assistant (1.0.10)
>     net.sf.opk.avro-schema-support (213.2.1)
>     net.seesharpsoft.intellij.plugins.csv (2.19.0)
>     com.mallowigi.idea (18.1.0)
>     org.jetbrains.kotlin (222-1.7.20-release-201-IJ4167.29)
>     com.github.copilot (1.1.33.1980)
>     atom-one (1.5.1)
>     String Manipulation (9.5.1)
>     org.sonarlint.idea (7.1.1.54565)
>     com.mallowigi (73.0.0)
>     Pythonid (222.4459.24)
>     org.intellij.scala (2022.2.13)
>     com.intellij.bigdatatools (222.3345.33)
>     izhangzhihao.rainbow.brackets.lite (1.0.0)
>     izhangzhihao.rainbow.brackets (2023.1.3-ij)
> 
> Kotlin: 222-1.7.20-release-201-IJ4167.29
> ```

After compare the env, I guess maybe caused by the themes, So I uninstall all the theme, but issue still exists, don't know what to do next...

**Comment by izhangzhihao** [comment]
Can you check the config again, make sure your settings are saved

<img width="705" alt="image" src="https://user-images.githubusercontent.com/122472234/217990210-03d74042-8f1c-438a-9f49-bf58d256fde8.png">

**Comment by izhangzhihao** [comment]
Hi, are you still having this issue?

**Comment by adongden** [comment]
> Hi, are you still having this issue?

Maybe I need to upgrade to latest idea 😊, I will reply after tried

**Comment by izhangzhihao** [comment]
I will convert this issue to a GitHub discussion, please continue to update the status in the discussion.

**Comment by izhangzhihao** [comment, marked as answer]
I think this question may be caused by the same reason as #2597： https://github.com/izhangzhihao/intellij-rainbow-brackets/issues/2597#issuecomment-1445290853
